### PR TITLE
Update cranmer.yml

### PR DESCRIPTION
**Pull request by cranmer**

### CODE REVIEW

**Comment by cranmer** [comment]
Not sure what's going wrong here... @henryiii ?

File: _data/people/cranmer.yml [hunk 1]
```diff
@@ -321,10 +321,24 @@ presentations:
     meetingurl: https://indico.cern.ch/event/962997/
     location: CERN (Virtual)
     focus-area: as
-  - title: How machine learning can help us get the most out of our highest fidelity physical models
-    date: 2020-10-21
-    url: https://www.jlab.org/AI/lunch_series
-    meeting: Jefferson Lab AI Lunch Series
-    meetingurl: https://www.jlab.org/AI/lunch_series
-    location: Jefferson Lab (Virtual)
+  - title: Vision: Physics, Machine Learning, and Computing
```
Comment (henryiii) [inline _data/people/cranmer.yml]: No-space colon space is illegal in yaml. Needs quotes.

File: _data/people/cranmer.yml [hunk 2]
```diff
@@ -321,10 +321,24 @@ presentations:
...
+    date: 2020-10-19
+    url: http://www.ncsa.illinois.edu/Conferences/AcceleratedAINCSA/agenda.html
+    meeting: 2020 Accelerated Artificial Intelligence for Big-Data Experiments Conference
+    meetingurl: http://www.ncsa.illinois.edu/Conferences/AcceleratedAINCSA/agenda.html
```
Comment (matthewfeickert) [inline _data/people/cranmer.yml]:
```suggestion
    url: https://www.ncsa.illinois.edu/Conferences/AcceleratedAINCSA/agenda.html
    meeting: 2020 Accelerated Artificial Intelligence for Big-Data Experiments Conference
    meetingurl: https://www.ncsa.illinois.edu/Conferences/AcceleratedAINCSA/agenda.html
```

Comment (matthewfeickert) [reply _data/people/cranmer.yml]: @cranmer Let's see if there's an HTTPS check failure.

Comment (matthewfeickert) [reply _data/people/cranmer.yml]: Okay seems no. :(

File: _data/people/cranmer.yml (hunk 1 above)
Comment (matthewfeickert) [inline _data/people/cranmer.yml]:
```suggestion
  - title: "Vision: Physics, Machine Learning, and Computing"
```

@cranmer noticed this `:` wasn't quoted.

Comment (matthewfeickert) [reply _data/people/cranmer.yml]: Good catch Kyle.

**Review by matthewfeickert — APPROVED** [review]
Passing CI now. :shipit:

**Comment by cranmer** [comment]
superseded by #488

**Review by henryiii — COMMENTED** [review]
Oops, meant to send this yesterday, didn't go through. Sorry!

**Pull request closed without merging**